lib/govuk_schemas/validator.rb

```diff
@@ -1,11 +1,13 @@
 module GovukSchemas
   class Validator
-    attr_reader :schema_name, :type, :payload
+    attr_reader :schema_name, :type
+    attr_accessor :payload
 
     def initialize(schema_name, type, payload)
       @schema_name = schema_name
       @type = type
       @payload = payload
+      ensure_payload_is_json
```

Comment:
It might be nicer to implement this as a method which returns a JSON string – rather than a method with side effects.

e.g.

```suggestion
      @payload = ensure_json(payload)
```

It feels a bit weird to be setting `@payload` and then immediately updating it in `ensure_payload_is_json`.

Reply:
hmm yea i think that's a good shout actually. Will update now 👍